Utilities/ocvalidate/ValidateKernel.c

```diff
@@ -215,39 +215,39 @@ CheckKernelAdd (
         "Kernel->Add[%u]->BundlePath (length %u) is too long (should not exceed %u)!\n",
         Index,
         AsciiStrLen (BundlePath),
-        OC_STORAGE_SAFE_PATH_MAX - L_STR_LEN (OPEN_CORE_KEXT_PATH)
+        OC_STORAGE_SAFE_PATH_MAX - L_STR_LEN (OPEN_CORE_KEXT_PATH) - 1
         ));
       ++ErrorCount;
     }
 
     //
     // There is one missing '\\' after the concatenation of BundlePath and ExecutablePath. Append one.
     //
-    ExecutableFixedSize   = L_STR_LEN (OPEN_CORE_KEXT_PATH) + AsciiStrLen (BundlePath) + 1;
-    ExecutablePathSumSize = ExecutableFixedSize + AsciiStrSize (ExecutablePath);
+    ExecutableFixedSize   = AsciiStrLen (BundlePath) + 1 + AsciiStrLen (ExecutablePath);
```

Comment:
I think `ExecutableFixedSize` is the wrong name here. In the original code it meant "the size of the part of the executable path which can't be avoided" (i.e. "Kexts\" + BundlePath). Also in the OC code Size strictly means the string size including the zero character, and Length means the string length not including the zero character. You're now using the var to hold the user string length (BundlePath + "\"+ ExecutablePath), so I suggest this var should be renamed ExecutableUserLength.